en/docs/integrations/integration-concepts.md

```diff
@@ -0,0 +1,98 @@
+   |                             | review_requested       | Request a user to review a pull request  |
+   |                             | review_request_removed |                                          |
+   |                             | labeled                | Add a label to a pull request            |
+   |                             | unlabeled              | Remove a label from a pull request       |
+   |                             | opened                 | Open a pull request                      |
+   |edited                       | Edit a pull request    |                                          |
+   |                             | closed                 | Close a pull request                     |
+   |                             | reopened               | Reopen a pull request                    |
+   | pull_request_review         | submitted              | Submit a pull request                    |
+   |                             | edited                 | Edit a pull request review               |
+   |                             | dismissed              | Dismiss a pull request review            |
+   | pull_request_review_comment | created                | Create a pull request review comment     |
+   |                             | edited                 | Edit a pull request review comment       |
+   |                             | deleted                | Delete a pull request review comment     | 
+
+
+## Connection
+
+A connection in Choreo allows you to connect to and interact with an external service through your application. You can use a connection to interact with generic protocol-based APIs and application specific APIs.
+
+Following are the connections you can use to interact with generic protocol-based APIs:
+  
+ - **HTTP**: Allows your application to interact with the HTTP API and communicate with an external endpoint via the HTTP protocol.   
+ - **SMTP**: Allows your application to send emails via the SMTP protocol.   
+ - **POP3**: Allows your application to receive emails from a POP3 email server.  
+ - **IMAP**: Allows your application to receive emails from any email server that supports the IMAP protocol. 
+
+Following are the connections you can use to interact with application specific APIs:
+
+ - **GitHub**: Allows your application to interact with the GitHub API and  perform required operations. 
+ - **Gmail**: Allows your application to interact with the Gmail API and perform operations such as creating, modifying, and sending emails.
+ - **Google Calendar**: Allows your application to interact with the Google Calendar API and perform required operations. 
+ - **Google Sheets**: Allows your application to interact with the Google Sheets API and perform operations such as creating and listing spreadsheets.
+ - **Twilio**: Allows your application to interact with the Twilio API and perform operations such as sending messages, WhatsApp messages, and making voice calls.
```

Comment:
Some connector types are missing, like slack, netsuite, salesforce etc

Reply:
Added the missing descriptions.